plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/resource/ResourcesService.java

```diff
@@ -307,8 +307,8 @@ private EntitySearchFilter createOrderFilter(RestListRequest requestList) {
             key = IResourceManager.RESOURCE_FILENAME_FILTER_KEY;
         }
 
-        EntitySearchFilter filter = new EntitySearchFilter(key, false);
-        filter.setOrder(requestList.getDirection().equals("desc") ?
+        EntitySearchFilter filter = new EntitySearchFilter(key, true);
```

Comment:
What does this change do?

Reply:
From the docs:
`Decide whether the filter is applied to an Entity Attribute (true) or to a metadata (false)`